apps/main/src/dex/hooks/useAutoRefresh.tsx

```diff
@@ -32,19 +33,20 @@ export const useAutoRefresh = (chainId: number | undefined) => {
     [chainId, fetchPoolsTvl, fetchPoolsVolume, poolDataMapper, setTokensMapper],
   )
 
+  const config = useConfig()
```

Comment:
argh this is only understandable by reading the import - it could be any config :disappointed:

Reply:
Can be removed when we migrate to hooks rather than imperative fetches 🤡